FabFos.py

```diff
@@ -215,6 +215,17 @@ def format(self, record):
 
 #################################### Classes end ################################################
 
+def bam2fastq(input_file, sample_id, output_dir, executables):
+    fastq_extract = [executables["bam2fastq"], "-o",
+                     output_dir + os.sep + sample_id + input_file + ".fasta", "--no-aligned", input_file]
```

Comment:
Typo in output path name - these should be ".fastq" not ".fasta". Also, let's use `_#` in the path in the same string as ".fastq" - `bam2fsatq` uses this to specify the forward and reverse reads.